packages/modeling/src/curves/bezier/create.js

```diff
@@ -0,0 +1,73 @@
+
+/**
+ * Creates an object representing a bezier easing curve. Can have both an arbitrary number of control points, and an arbitrary number of dimensions.
+ *
+ * @example
+ * const b = bezier.create([0,10]) // a linear progression from 0 to 10
+ * const b = bezier.create([0, 0, 10, 10]) // a symmetrical cubic easing curve that starts slowly and ends slowly from 0 to 10
+ * const b = bezier.create([0,0,0], [0,5,10], [10,0,-5], [10,10,10]]) // a cubic 3 dimensional easing curve that can generate position arrays for modelling
+ *
+ * @param {Array} points An array with at least 2 elements of either all numbers, or all arrays of numbers that are the same size.
+ * @returns {bezier} a new bezier data object
+ * @alias module:modeling/curves/bezier.create
+ */
+const create = (points) => {
```

Comment:
How about using rest parameters here? Then it's possible to create using just a series of numbers or a series of arrays.
```
const b = bezier.create( 0,10 ) // a linear progression
const b = bezier.create( 0, 0, 10, 10 ) // a symmetrical cubic easing curve
const b = bezier.create( [0,0,0], [0,5,10], [10,0,-5], [10,10,10] ) // a cubic 3 dimensional easing curve
```
Here's a good explanation of the rest parameters. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

Reply:
Sure.  I wasn't sure if that was a JSCAD thing for creation objects.  Most analogous would be slice/fromPoints, which does not offer this.  if this is a direction you want jscad to go in, I'm happy to do it though.

Reply:
JSCAD API for transforms, booleans, etc. are already using rest parameters, so no issues.

Try it out. I think it makes sense.

Reply:
Sure, but those are saying "Apply this to these independent entities".  

bezier.create(), slice.fromPoints(), geom2.fromPoints(), etc are all "create something from this set of tightly related data".  I would argue that it makes sense not to use rest parameters for those, especially if the data set that you are passing in may be generated algorithmically.

Reply:
Sounds fine then. Let’s stay with the current parameters.